framework/tests/accumulation/Generics.java

```diff
@@ -0,0 +1,45 @@
+// Copied from the Object Construction Checker.
```

Comment:
Please remove these comments, or indicate how they are actionable.  For example, is the programmer supposed to keep the two versions of the test in sync by making a duplicate pull request on the other repository?